### DISCUSSION TITLE
zigbee2mqtt problema con pairing

### DISCUSSION THREAD

**Discussion by kimuss** [indomus/forum#2516] [ZigBee, Z-Wave, Thread, Matter]
salve a tutti, 
dopo aver cambiato canale non riesco più a fare il pairing dei dispositivi.
non riesco a capire il problema dai logs e chiedo una mano, sicuramente sbaglio qualcosa.
il problema è che semplicemente, mettendo in pairing un dispositivo (lampeggia come scritto nel manuale, quindi è sicuramente in pairing), zigbee2mqtt  non scrive nulla... nell'interfaccia web non appare nulla..

sono in ambiete docker su rasp, conbee2

log:

```

Zigbee2MQTT:info  2023-06-24 18:54:25: Logging to console and directory: '/var/log' filename: log.txt
Zigbee2MQTT:info  2023-06-24 18:54:25: Starting Zigbee2MQTT version 1.31.2 (commit #21f5125)
Zigbee2MQTT:info  2023-06-24 18:54:25: Starting zigbee-herdsman (0.14.117)
Zigbee2MQTT:info  2023-06-24 18:54:26: zigbee-herdsman started (resumed)
Zigbee2MQTT:info  2023-06-24 18:54:26: Coordinator firmware version: '{"meta":{"maintrel":0,"majorrel":38,"minorrel":120,"product":0,"revision":"0x26780700","transportrev":0},"type":"ConBee2/RaspBee2"}'
Zigbee2MQTT:info  2023-06-24 18:54:26: Currently 0 devices are joined:
Zigbee2MQTT:info  2023-06-24 18:54:26: Zigbee: disabling joining new devices.
Zigbee2MQTT:info  2023-06-24 18:54:26: Connecting to MQTT server at mqtt://192.168.1.1
Zigbee2MQTT:info  2023-06-24 18:54:26: Connected to MQTT server
Zigbee2MQTT:info  2023-06-24 18:54:26: MQTT publish: topic 'zigbee2mqtt/bridge/state', payload 'online'
Zigbee2MQTT:info  2023-06-24 18:54:27: Started frontend on port 0.0.0.0:8080
Zigbee2MQTT:info  2023-06-24 18:54:27: MQTT publish: topic 'zigbee2mqtt/bridge/config', payload '{"commit":"21f5125","coordinator":{"meta":{"maintrel":0,"majorrel":38,"minorrel":120,"product":0,"revision":"0x26780700","transportrev":0},"type":"ConBee2/RaspBee2"},"log_level":"info","network":{"channel":25,"extendedPanID":"0xdddddddddddddddd","panID":6754},"permit_join":false,"version":"1.31.2"}'
Zigbee2MQTT:info  2023-06-24 18:54:27: MQTT publish: topic 'zigbee2mqtt/bridge/state', payload 'online'
Zigbee2MQTT:info  2023-06-24 18:54:27: Zigbee2MQTT started!
Zigbee2MQTT:info  2023-06-24 18:54:57: Zigbee: allowing new devices to join.
Zigbee2MQTT:info  2023-06-24 18:54:57: MQTT publish: topic 'zigbee2mqtt/bridge/response/permit_join', payload '{"data":{"time":254,"value":true},"status":"ok","transaction":"btefy-1"}'
```
ho già provato varie strade cancellando db, riavviando, togliendo alimentazione etc... 
grazie per l'attenzione

**Comment by snakuzzo** [comment]
Non riesci a capire il problema dai log perché di fatto non viene loggato alcun problema

**Comment by kimuss** [comment]
infatti... 

domanda, volevo provare a mettere in pairing la chiavetta sonoff ZBDongle-E per provare a capire se la vede (flashata tramite vostra guida), ma non so come metterla in pairing
sapete come va in pairing? ho provato il tasto reset in varie salse ma non riesco

**Reply by snakuzzo** [reply]
Scusa ma cosa stai facendo ?
Chi dovrebbe vedere cosa ?
La ZBDongle-E flashata con firmware coordinator fa lei da coordinator e sostituirebbe il tuo attuale coordinator

**Reply by kimuss** [reply]
nono è come router, scusa non avevo precisato

**Reply by snakuzzo** [reply]
In teoria non dovresti fare nulla.
Abiliti il pairing su zigbee2mqtt e la alimenti (senza premere alcun tasto), assicurandoti che si vicino al coordinator

**Comment by kimuss** [comment]
allora ho provato a mettere la conbee su un piccolo hub per usb, e adesso conn i vari sensori etc riesce a fare il pairing, mi rimane la usb della sonoff che non andando in pairing non si connette, devo riflashare il firmware?

**Reply by snakuzzo** [reply]
e perché dovresti riflashare ? Non è che "de botto" si sfascia il firmware.
Sinceramente, non abbiamo mai provato ad utilizzarla come router, quindi non saprei dirti che fare.
Prova a guardare sul progetto di zigbee2mqtt e...una cortesia...non creare ogni volta nuovi thread in questa discussione. Rispondi sotto l'ultimo messaggio inviato. Grazie.

**Reply by kimuss** [reply]
ok grazie, ho creato nuovi thread perché la prima risposta inviata l'hai fatta come nuovo thread, pensavo fosse giusto fare dei comment e non dei reply, visto che dovevo segnalare nel primo comment il problema del pairing della sonoff, e nel secondo comment volevo segnalare la possibile risoluzione usando un hub, errore mio, grazie potete chiudere

**Reply by snakuzzo** [reply]
La prima risposta è sempre un nuovo thread
